ci/steps.yml

```diff
@@ -175,13 +175,34 @@ steps:
     displayName: Build accelbubble example application to test for android
 
   ##----------------------------------------------------
-  # determine Windows build system
+  # Cache Powershell Modules in $MODULES_FOLDER
+  - task: Cache@2
+    inputs:
+      key: '"pwsh modules" | "$(Agent.OS)" | "$(Agent.Name)" | "$(Agent.Version)"'
+      # Agent.Version will invalidate the cache every few weeks, when a new agent is released
```

Comment:
`"$(Agent.Name)" | "$(Agent.Version)"` are the wrong properties to use here. I was hoping to use a variable that evaluates to `windows-2019`, and a version number for the VM image, but I have not found one.